### PR TITLE
Feature 44 employer profile 

### PR DESCRIPTION
**Pull request by Pchelnikova**

### CODE REVIEW

File: JobHunter.Presentation/ClientApp/src/app/app.module.ts [hunk 1]
```diff
@@ -17,10 +17,12 @@ import { appRoutes } from './route.routing';
 import { SpinnerComponent } from './components/spinner/spinner.component';
 import { HomeComponent } from './components/home/home.component';
 
+
```
Comment (katemalash) [inline JobHunter.Presentation/ClientApp/src/app/app.module.ts]: Could you please explain what does this empty string do here?

File: JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts [hunk 2]
```diff
@@ -16,19 +16,28 @@ import { CurrencyPipe } from '@angular/common';
...
+    console.log(this.router);
   }
   
   addPhone() {
-    this.phones.push({number:'+38'});
+    this.phones.push({number_phone:'+38'});
   }
 
+  canselAddPhone() {
+
```
Comment (katemalash) [inline JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts]: Could you please explain what does this empty string do here?

File: JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts [hunk 3]
```diff
@@ -16,19 +16,28 @@ import { CurrencyPipe } from '@angular/common';
...
+    this.phones.push({number_phone:'+38'});
   }
 
+  canselAddPhone() {
+  
+    this.phones.pop();
+    var len = this.phones.length;
+    console.log( length = len);
+  }
+
```
Comment (katemalash) [inline JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts]: Could you please explain what does this empty string do here?

File: JobHunter.Presentation/ClientApp/src/app/app.module.ts [hunk 4]
```diff
@@ -48,13 +50,15 @@ export function tokenGetter() {
     HotVacanciesComponent,
     CompaniesListComponent,
     HomeComponent,
+
```
Comment (katemalash) [inline JobHunter.Presentation/ClientApp/src/app/app.module.ts]: Could you please explain what does this empty string do here?

File: JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts (hunk 2 above)
Comment (katemalash) [inline JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts]: Could you please explain what does this empty string do here?

Comment (katemalash) [inline JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts]: Could you please explain what does this empty string do here?

File: JobHunter.Presentation/ClientApp/src/app/app.module.ts (hunk 1 above)
Comment (Pchelnikova) [reply JobHunter.Presentation/ClientApp/src/app/app.module.ts]: Its superfluous here. will be removed next

File: JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts (hunk 2 above)
Comment (Pchelnikova) [reply JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts]: Its superfluous here. will be removed next

File: JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts (hunk 3 above)
Comment (Pchelnikova) [reply JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts]: Its superfluous here. will be removed next

File: JobHunter.Presentation/ClientApp/src/app/app.module.ts (hunk 4 above)
Comment (Pchelnikova) [reply JobHunter.Presentation/ClientApp/src/app/app.module.ts]: Its superfluous here. will be removed next

File: JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts (hunk 2 above)
Comment (Pchelnikova) [reply JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts]: Its superfluous here. will be removed next

Comment (Pchelnikova) [reply JobHunter.Presentation/ClientApp/src/app/components/employer-profile/employer-profile.component.ts]: Its superfluous here. will be removed next

**Pull request merged**